composer.json

```diff
@@ -14,7 +14,8 @@
     "ocramius/proxy-manager": "2.*",
     "psr/log": "1.*",
     "psr/cache": "1.0.*",
-    "doctrine/annotations": "1.3.*"
+    "doctrine/annotations": "1.3.*",
+    "symfony/expression-language": "3.*"
   },
```

Comment:
you can put it in both require-dev (so you can test it) and in suggests. 
Who requires usage of expression should include the package explictly